src/itest/java/uk/gov/companieshouse/insolvency/data/steps/InsolvencySteps.java

```diff
@@ -70,20 +70,19 @@ public void the_expected_result_should_match(String string) throws IOException {
 
         Assertions.assertThat(insolvencyDocuments).hasSize(1);
 
-        InsolvencyDocument actual = insolvencyRepository.findByCompanyNumber(this.companyNumber);
+        String uri = "/company/{company_number}/insolvency";
+        String companyNumber = "CH5324324";
```

Comment:
I think lets keep old step as it was, and write a new feature file to test GET endpoint on its own and assert GET is fetching data correctly from DB